test/runner/browser/runner_test.dart

```diff
@@ -25,6 +25,16 @@ void main() {
 }
 """;
 
+final _importFailure = """
+import 'package:test/test.dart';
+import 'blah' as bad_library;
+
+void main() {
+  bad_library.main();
+  test("failure", () => throw new TestFailure("oh no"));
+}
+""";
```

Comment:
Since this is only used in one test, just include it inline.

Also, add a comment mentioning why bad imports cause this error.